scripts/tests/chiptest/__init__.py

```diff
@@ -27,6 +30,164 @@
+                yield ManualTest(yaml="%s.yaml" % name, reason=json_file_path)
+
+
+def _GetManualTests() -> Set[ManualTest]:
+    manualtests = set()
+
+    # TODO:
+    #
+    # These are NOT manual tests, but rather "tests that fail in yaml and
+    # for this reason are marked as manual".
+    #
+    # We are working to get this list down to 0.
+    manualtests.add(ManualTest(
+        yaml="OTA_SuccessfulTransfer.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_ACE_1_1.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_ACE_1_5.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_ACL_2_10.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_ACL_2_7.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_ACL_2_8.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_ACL_2_9.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_ACT_3_1_Simulated.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_APBSC_9_1.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_BINFO_2_1.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_BINFO_2_3_Simulated.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_BOOL_3_1_Simulated.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_CC_6_2.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_CC_6_3.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_CC_8_1.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_CGEN_2_1.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_CHANNEL_5_3.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_DESC_2_2_Simulated.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DGETH_2_1.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DGETH_2_2.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_DGETH_3_1_Simulated.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DGGEN_2_1.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_DGSW_3_1_Simulated.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DGWIFI_2_1.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_DGWIFI_3_1_Simulated.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DRLK_2_2.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DRLK_2_3.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DRLK_2_5.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DRLK_2_7.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_FLABEL_3_1_Simulated.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_FLW_3_1_Simulated.yaml", reason="TODO"))
```

Comment:
`*_Simulated` tests are not manual tests. Those are tests used by the placeholder apps.
It is unclear that you will be able to run those using `chip-repl` currently.